engine-cdi/pom.xml

```diff
@@ -27,7 +27,7 @@
 
   <profiles>
     <profile>
-      <id>jdk11-and-onwards</id>
```

Comment:
- ❓ I like the `-and-onwards` notation. Why did you change it?

Reply:
Mainly due to naming consistency, actually. We have other profiles that imply "from here on and upwards" like some JDK 9 profiles. This is the only reason, really.